keyboards/planck/keymaps/pascamel/config.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you remove the ifndef and define here, and replace it with `#pragma once`, and remove the endif at the end of the file?

Reply:
@drashna updated as requested!